src/routes/templates/+page.svelte

```diff
@@ -1,6 +1,88 @@
 <script lang="ts">
-  import Templates from "$lib/components/Templates.svelte";
+  import { goto, invalidateAll } from '$app/navigation';
+    
+  export let data:any;
+  
+  const deleteRequest = async (id:string):Promise<void> => {
+    const res = await fetch(`https://test.axelzublena.com/api/templates/${id}`, {
```

Comment:
Nitpick but could be a good thing for response variables to be named the same everywhere. So "response" instead of "res".